src/ontology/mondo-edit.obo

```diff
@@ -560385,6 +560385,63 @@ name: arterial tortuosity-bone fragility syndrome
 xref: OMIM:620908 {source="MONDO:equivalentTo"}
 is_a: MONDO:0002254 {source="OMIM:620908", source="https://orcid.org/0000-0002-4142-7153"} ! syndromic disease
 
+[Term]
+id: MONDO:0975751
+name: brain abnormalities-severe developmental delay-facial dysmorphism-intellectual disability syndrome due to MEF2C mutation
+xref: Orphanet:664416 {source="MONDO:equivalentTo"}
+is_a: MONDO:0002254 {source="https://orcid.org/0000-0002-4142-7153"} ! syndromic disease
+relationship: excluded_subClassOf MONDO:0013266 {source="Orphanet:664416", source="https://orcid.org/0000-0002-4142-7153"} ! intellectual disability, autosomal dominant 20
+
+[Term]
+id: MONDO:0975752
+name: littoral cell hemangioma of the spleen
+xref: Orphanet:673538 {source="MONDO:equivalentTo"}
+is_a: MONDO:0024296 {source="PMID:31971725", source="https://orcid.org/0000-0002-4142-7153"} ! vascular neoplasm
+relationship: excluded_subClassOf MONDO:0971115 {source="Orphanet:673538", source="https://orcid.org/0000-0002-4142-7153"} ! benign vascular tumor
+
+[Term]
+id: MONDO:0975753
+name: papillary hemangioma
+xref: Orphanet:673543 {source="MONDO:equivalentTo"}
+is_a: MONDO:0971115 {source="Orphanet:673543", source="PMID:17927585", source="https://orcid.org/0000-0002-4142-7153"} ! benign vascular tumor
+
+[Term]
+id: MONDO:0975754
+name: pseudomyogenic hemangioendothelioma
+xref: Orphanet:673556 {source="MONDO:equivalentTo"}
+is_a: MONDO:0971116 {source="Orphanet:673556", source="PMID:36108375", source="https://orcid.org/0000-0002-4142-7153"} ! borderline vascular tumor
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
```

Comment:
Orphanet has several classes classified under 'borderline vascular tumor':
https://www.orpha.net/en/disease/classification/group/162973?orphaCode=673473&firstParent=0&name=Borderline-vascular-tumor&mode=name
![image](https://github.com/user-attachments/assets/2ba0bf5b-33fd-413c-a311-f780ded55257)

I wonder if we should classify these other cancers under this class? Or maybe this is not a good grouping?

NCIT has a similar term, Intermediate Vascular Neoplasm, and has their own classification, maybe we want to follow this:
https://www.ebi.ac.uk/ols4/ontologies/ncit/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FNCIT_C7334

![image](https://github.com/user-attachments/assets/99ae2521-1480-4254-882e-1efbda655a95)

I don't think this should be a blocker for this PR, but if you think we should consider this classification, we should create a ticket.

Reply:
Thank you @nicolevasilevsky. 
I think we should definitely review classification. I will create an issue for this, as I think it will take some time to review. (note: I am hoping that we can get a lot of these classifications with the subclass alignment)